client/signup/steps/domains/index.jsx

```diff
@@ -145,6 +148,34 @@ class DomainsStep extends Component {
 		return isPlansStepExistsInFutureOfFlow && ! isPlanStepSkipped;
 	};
 
+	componentDidMount() {
+		const signupFlows = [
+			'onboarding',
```

Comment:
Would you also like to include the `launch-site` flow?

Reply:
Added it!